### DISCUSSION TITLE
Prefix Run Test command

### DISCUSSION THREAD

**Discussion by renatorfr** [elixir-lsp/vscode-elixir-ls#220] [Ideas]
Hi,

I'm using ElixrLS and I like the feature "Run test" over test cases but in our project we need to set some env vars before running the tests.
It would be nice to have a way to config a prefix command to `mix test`, this way we could set in configuration something like `FOO=bar` and the final run command would be `FOO=bar && mix test ...`.

**Comment by lukaszsamson** [comment]
Isn’t it already covered by https://github.com/elixir-lsp/vscode-elixir-ls/pull/213?

**Reply by renatorfr** [reply]
Hum... I think partly, yes.

We use [nv](https://github.com/jcouture/nv) with `.env` files, so I was thinking about `nv .env` as a prefix.
This gives us a single place to add/change env vars.

Maybe a config where we could set a file with the env vars we want to set?
